### PR TITLE
Remove Android 12 Spooky Warning

### PR DESCRIPTION
**Pull request by nathan-contino-mongo**
### Jira

- Conversations in realm-java-team slack channel

### Staged Changes (Requires MongoDB Corp SSO)

- [Android - React to Changes](https://docs-mongodbcom-staging.corp.mongodb.com/realm/docsworker-xlarge/android-12-warning-removal/sdk/android/examples/react-to-changes/)
- [Android - Troubleshooting](https://docs-mongodbcom-staging.corp.mongodb.com/realm/docsworker-xlarge/android-12-warning-removal/sdk/android/troubleshooting/#change-listeners-in-android-12-with-sdk-versions-below-10.5.1)

### Review Guidelines

[REVIEWING.md](https://github.com/mongodb/docs-realm/blob/master/REVIEWING.md)

### CODE REVIEW

**Review by cmelchior — APPROVED** [review]

File: source/sdk/android/troubleshooting.txt [hunk 1]
```diff
@@ -62,11 +62,16 @@ the ``REALM_DISABLE_ANALYTICS`` environment variable to ``true``.
 Change Listeners in Android 12 with SDK Versions Below 10.5.1
 -------------------------------------------------------------
 
-Due to an update to Linux kernel 5.x,
+Due to a change in the Linux kernel,
 :ref:`object, collection, and {+realm+} notifications
 <android-client-notifications>` do not work in SDK versions below
-10.5.1 on emulated devices running Android 12. For full functionality
-in Android 12, update to SDK version 10.5.1.
+10.5.1 on certain emulated devices running early versions of
```
Comment (nathan-contino-mongo) [inline source/sdk/android/troubleshooting.txt:68]:
```suggestion
10.5.1 on devices running certain early versions of
```

File: source/sdk/android/troubleshooting.txt [hunk 2]
```diff
@@ -62,11 +62,16 @@ the ``REALM_DISABLE_ANALYTICS`` environment variable to ``true``.
...
 <android-client-notifications>` do not work in SDK versions below
-10.5.1 on emulated devices running Android 12. For full functionality
-in Android 12, update to SDK version 10.5.1.
+10.5.1 on devices running certain early versions of
+Android 12.
+
+This change effects Linux kernel versions beginning with `5.5`.
+Linux kernel version `5.14-rc4` fixed the issue. The fix was
+also backported to `LTS 5.10.56` and `LTS 5.13.8`. All mainline
+and LTS Android 12 branches contain the fix or a backport of it.
```
Comment (mongodben) [inline source/sdk/android/troubleshooting.txt:74]: is there any CTA for what the user can do to resolve the issue? 

like how in the previous version it said "update to SDK version 10.5.1"?

Comment (nathan-contino-mongo) [reply source/sdk/android/troubleshooting.txt:74]:
```suggestion
and LTS Android 12 branches contain the fix or a backport of it.

If you experience this issue, you can restore notification functionality
with the following fixes:

- upgrade to a version of the SDK later than 10.5.1.
- upgrade to a version of Android 12 that uses a Linux kernel release
   that contains the fix (kernel commit `3a34b13a88caeb2800ab44a4918f230041b37dd9`)
   or the backport of the fix (kernel commit `4b20d2de0b367bca627b49efd8d2e9e01bb66753`).
```

File: source/sdk/android/troubleshooting.txt [hunk 3]
```diff
@@ -62,11 +62,24 @@ the ``REALM_DISABLE_ANALYTICS`` environment variable to ``true``.
...
+
+If you experience this issue, you can restore notification functionality
+with the following fixes:
+
+- upgrade to a version of the SDK later than 10.5.1.
+- upgrade to a version of Android 12 that uses a Linux kernel release
+   that contains the fix (kernel commit `3a34b13a88caeb2800ab44a4918f230041b37dd9`)
+   or the backport of the fix (kernel commit `4b20d2de0b367bca627b49efd8d2e9e01bb66753`).
```
Comment (nathan-contino-mongo) [inline source/sdk/android/troubleshooting.txt:82]:
```suggestion
  that contains the fix (kernel commit `3a34b13a88caeb2800ab44a4918f230041b37dd9`)
  or the backport of the fix (kernel commit `4b20d2de0b367bca627b49efd8d2e9e01bb66753`).
```

File: source/sdk/android/troubleshooting.txt [hunk 4]
```diff
@@ -62,11 +62,24 @@ the ``REALM_DISABLE_ANALYTICS`` environment variable to ``true``.
...
+Linux kernel version `5.14-rc4` fixed the issue. The fix was
+also backported to `LTS 5.10.56` and `LTS 5.13.8`. All mainline
+and LTS Android 12 branches contain the fix or a backport of it.
+
+If you experience this issue, you can restore notification functionality
+with the following fixes:
+
+- upgrade to a version of the SDK later than 10.5.1.
+- upgrade to a version of Android 12 that uses a Linux kernel release
+  that contains the fix (kernel commit `3a34b13a88caeb2800ab44a4918f230041b37dd9`)
```
Comment (mongodben) [inline source/sdk/android/troubleshooting.txt:81]:
```suggestion
  that contains the fix (kernel commit ``3a34b13a88caeb2800ab44a4918f230041b37dd9``)
```

File: source/sdk/android/troubleshooting.txt [hunk 5]
```diff
@@ -62,11 +62,24 @@ the ``REALM_DISABLE_ANALYTICS`` environment variable to ``true``.
...
+  or the backport of the fix (kernel commit `4b20d2de0b367bca627b49efd8d2e9e01bb66753`).
```
Comment (mongodben) [inline source/sdk/android/troubleshooting.txt:82]:
```suggestion
  or the backport of the fix (kernel commit ``4b20d2de0b367bca627b49efd8d2e9e01bb66753``).
```

**Review by mongodben — APPROVED** [review]
2 tiny rst-y things otherwise lgtm! good addition

File: source/sdk/android/troubleshooting.txt [hunk 6]
```diff
@@ -62,11 +62,24 @@ the ``REALM_DISABLE_ANALYTICS`` environment variable to ``true``.
...
 <android-client-notifications>` do not work in SDK versions below
-10.5.1 on emulated devices running Android 12. For full functionality
-in Android 12, update to SDK version 10.5.1.
+10.5.1 on devices running certain early versions of
+Android 12.
+
+This change effects Linux kernel versions beginning with `5.5`.
+Linux kernel version `5.14-rc4` fixed the issue. The fix was
+also backported to `LTS 5.10.56` and `LTS 5.13.8`. All mainline
+and LTS Android 12 branches contain the fix or a backport of it.
```
Comment (nathan-contino-mongo) [inline source/sdk/android/troubleshooting.txt:74]:
```suggestion
This change effects Linux kernel versions beginning with ``5.5``.
Linux kernel version ``5.14-rc4`` fixed the issue. The fix was
also backported to ``LTS 5.10.56`` and ``LTS 5.13.8``. All mainline
and LTS Android 12 branches contain the fix or a backport of it.
```

**Pull request merged**